src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherEmployeeInformationValidation.java

```diff
@@ -24,7 +24,8 @@
 public class CuDisbursementVoucherEmployeeInformationValidation extends DisbursementVoucherEmployeeInformationValidation {
 	
     private static final Logger LOG = LogManager.getLogger(CuDisbursementVoucherEmployeeInformationValidation.class);
-
+    private DataDictionaryService dataDictionaryService;
+    private PersonService personService;
```

Comment:
Can these private variables be removed (along with the setter overrides)? I think we've overlayed the superclass so that its versions of these variables will have "protected" scope.

Reply:
updated